docs/references/phpdoc/tags/since.rst

```diff
@@ -1,39 +1,43 @@
 @since
 ======
 
-.. important::
-
-   The effects of this tag are not yet fully implemented in phpDocumentor 3.
-
-The @since tag indicates at which version did the associated
-Structural Elements became available.
+The ``@since`` tag is used to denote _when_ an element was introduced or modified.
 
 Syntax
 ------
 
-    @since [version] [<description>]
+.. code-block::
+
+    @since [<Semantic Version>] [<description>]
```

Comment:
Given the many cases of SHOULD/MAY flexibility we _have_ to allow for many tags, I'd say this should remain just `[version]`, to account for the many places where a version# is used that doesn't actually qualify as _Semantic_.

This is in line with the use mentioned just below, where _Semantic_ is `RECOMMENDED` rather than _required_.  If we chose to _require_ it, any projects using non-Semantic for their own reasons (many historical) would not be PSR compliant, if only because of this one requirement.

Reply:
Fixed